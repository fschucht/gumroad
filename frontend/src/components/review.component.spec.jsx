import renderer from 'react-test-renderer';
import { Review } from './review.component';

describe('Review', () => {
  it('should render a Review component', () => {
    const renderedComponent = renderer
      .create(
        <Review review={{
          rating: 5,
          comment: 'Comment',
          user: {
            email: 'email'
          }
        }}>Hello World!</Review>
      )
      .toJSON()

    expect(renderedComponent).toMatchSnapshot()
  })
})