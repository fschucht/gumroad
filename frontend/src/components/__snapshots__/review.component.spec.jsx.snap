// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Review should render a Review component 1`] = `
Array [
  <div
    className="view view--layout-row-space-between-center"
  >
    <p
      className="text"
    >
      email
    </p>
    <div
      className="rating"
    >
      <svg
        className="rating__star rating__star--active"
      >
        star.svg
      </svg>
      <svg
        className="rating__star rating__star--active"
      >
        star.svg
      </svg>
      <svg
        className="rating__star rating__star--active"
      >
        star.svg
      </svg>
      <svg
        className="rating__star rating__star--active"
      >
        star.svg
      </svg>
      <svg
        className="rating__star rating__star--active rating__star--last"
      >
        star.svg
      </svg>
    </div>
  </div>,
  <div
    className="view view--with-border-bottom"
  >
    <p
      className="text"
    >
      Comment
    </p>
  </div>,
]
`;
